src/module-vsbridge-indexer-core/etc/adminhtml/system.xml

```diff
@@ -121,6 +121,33 @@
                     <comment>in seconds</comment>
                 </field>
             </group>
+
+            <group id="optimization_settings" translate="label" type="text" sortOrder="350" showInDefault="1" showInWebsite="0" showInStore="0">
+                <label>Optimization Settings</label>
+                <depends>
+                    <field id="vsbridge_indexer_settings/general_settings/enable">1</field>
+                </depends>
+                <field id="cluster_health" translate="label comment" type="select" sortOrder="50" showInDefault="1" showInWebsite="0" showInStore="0">
+                    <label>Check cluster health</label>
+                    <comment>Set yes if you want to check before sending operation to update if status is green</comment>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
```

Comment:
I just wonder if we should make it optional? It probably is best to always check this.. but I think  we should also ask @jonathanribas

Reply:
I think by default it should be set to **Yes**.

Reply:
If we talk about `number_of_replicas` and `refresh_interval` maybe it should be **Yes** by default but i'm afraid what will happen when sth/sb force stopping indexing and this settings not return to previous values. IMO this solution is good for people who know ES.

If we talk about checking cluster status i think that a lot of instances has yellow status all the time, for example if they use only one node. And again i'm afraid about people with low knowledge about ES.

What do you think about my fears? It's no problem to change it to **Yes** by default but i don't know is it good idea.